src/Interpreters/SystemLog.h

```diff
@@ -325,9 +340,16 @@ void SystemLog<LogElement>::savingThreadFunction()
             uint64_t to_flush_end = 0;
 
             {
+                LOG_TRACE(log, "Sleeping");
```

Comment:
This (and the `Woke up` too) is pretty verbose logging, will it be reverted after the problem will be debugged?

Reply:
Yes, I'll try to make it less verbose. Although the unfortunate reality is that if you had to add a log message once, you'll probably need it again. Maybe we have to add an even deeper log level...